DuckDuckGo/UserText.swift

```diff
@@ -1144,4 +1144,26 @@ But if you *do* want a peek under the hood, you can find more information about
     public static let brokenSiteReportToggleAlertYesButton = NSLocalizedString("broken.site.report.toggle.alert.yes.button", value: "Yes", comment: "Button to answer question 'Did turning off protections resolve the issue on this site?'")
     public static let brokenSiteReportToggleAlertNoButton = NSLocalizedString("broken.site.report.toggle.alert.no.button", value: "No", comment: "Button to answer question 'Did turning off protections resolve the issue on this site?'")
 
+    // MARK: -  Dax Onboarding Experiment
```

Comment:
Thanks for adding these proactively, but they will end up in the text to be localised if someone does a string export and we're not ready for that yet. Localisation should be done immediately after the ship review (or even as a follow up). 

I think a quick way to do this is to add a file private function with the same signature to UserText.swift (probably near the top so people can see it and re-reuse it), e.g. 

    /// Use this to add strings you're not ready to localize yet.  When you are, switch back to NSLocalizedString init.
    fileprivate func NonLocalizedString(_ key: String, value: String, comment: String) -> String {
        return value
    }

Then you can just use `NonLocalizedString` and when we're ready to ship rename those to `NSLocalizedString`.

(I thought there was something similar in the codebase already but I can't see it)

Reply:
Hey @brindy no problems. I’m not merging this PR to `main` but to the main feature branch `alessandro/onboarding-intro` so technically there’s no risk. 

Having said that I can add that function if you prefer.

Reply:
I think it’s ok to leave in since it’s not going to main… 
also since the copy is already approved and used on Android I think we could also get the start getting the translations to save time (hopefully we can reuse the Android ones, Smartling should be able to).

Reply:
Fair points!